styles.tsx

```diff
@@ -0,0 +1,35 @@
+import { StyleSheet } from "react-native";
+
+const style = StyleSheet.create({
+  container: {
+    backgroundColor: "#FFF7E6",
+    paddingVertical: 50,
+    paddingHorizontal: 20,
+    justifyContent: "center",
+  },
+  buttonStyle: {
+    // TODO: Add generic button style for touchable opacity component
+    backgroundColor: "#FFC1C1",
```

Comment:
Remove comments.

Reply:
I have removed the comments in the recent commit.